source/slang/slang-compiler.h

```diff
@@ -2733,11 +2733,14 @@ namespace Slang
 
         SlangResult emitPrecompiledDXIL(ComPtr<IArtifact>& outArtifact);
 
+        SlangResult emitPrecompiledSPIRV(ComPtr<IArtifact>& outArtifact);
+
         void maybeDumpIntermediate(IArtifact* artifact);
 
-        // Used to cause instructions available in precompiled DXIL to be
+        // Used to cause instructions available in precompiled blobs to be
         // removed between IR linking and target source generation.
         bool removeAvailableInDXIL = false;
+        bool removeAvailableInSPIRV = false;
```

Comment:
can we unify them as `removeAvailableInDownstreamIR`?

Reply:
I don't think so because one module IR may have both embedded DXIL and embedded SPIRV, with a different assortment of coverage in either precompiled blob.

Reply:
But when you are compiling for dxil, you obviously want to remove things that are availble in dxil if the flag is set, and same for spirv. There is no need to think about dxil when generating spirv or vice versa.

Reply:
Okay, done.